packages/OCADml/OCADml.0.1.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Types and functions for building CAD packages in OCaml"
+description: "Types and functions for building CAD packages in OCaml"
+maintainer: ["Geoff deRosenroll<geoffderosenroll@gmail.com"]
+authors: ["Geoff deRosenroll<geoffderosenroll@gmail.com"]
+license: "GPL-2.0-or-later"
+homepage: "https://github.com/OCADml/OCADml"
+doc: "https://OCADml.github.io/OCADml"
+bug-reports: "https://github.com/OCADml/OCADml/issues"
+depends: [
+  "dune" {>= "3.3"}
+  "ocaml" {>= "4.08.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.13.0"}
```
```
#=== ERROR while compiling OCADml.0.1.0 =======================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.12.1 | pinned(https://github.com/OCADml/OCADml/releases/download/v0.1.0/OCADml-0.1.0.tbz)
# path                 ~/.opam/4.12/.opam-switch/build/OCADml.0.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p OCADml -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/OCADml-8-28e153.env
# output-file          ~/.opam/log/OCADml-8-28e153.out
### output ###
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.OCADml.objs/byte -I /home/opam/.opam/4.12/lib/cairo2 -no-alias-deps -open OCADml__ -o lib/.OCADml.objs/byte/oCADml__Util.cmo -c -impl lib/util.ml)
# File "lib/util.ml", line 194, characters 19-26:
# 194 |   let min_slices = Int.max 1 Float.(twist /. (pi /. 3.) |> ceil |> to_int) in
#                          ^^^^^^^
# Error: Unbound value Int.max
```

Reply:
Oh interesting, I thought the `4.08.x` and `4.13.x` tests that I do in the CI of a dependent library would have caught this problem. I'm not attached to the old compiler support though, so I'll just bump it I guess.